### ISSUE TITLE
synchronization: qc rule "lc_o_1" throws a lot of errors

### ISSUE THREAD

**Issue by lbestock** [arch-kiosk/arch-kiosk-office#2914]
I am not used to getting scary housekeeping errors. Does this represent a problem?
![image](https://github.com/user-attachments/assets/89e3ca24-c1d0-49fc-91ac-31bdfac6efec)
For context, I have deleted a unit and attempted to delete a site before this synchronization, and changed some configuration settings.

**Comment by urapadmin** [comment]
no, that affects quality control only. But I will look into it.

**Comment by urapadmin** [comment]
that's a real bug in quality control. Took me a while to find it. 
It only occurs in projects that have the qc rule "lc_o_1" activated. It tests if a locus of type "other" has a type set in the detail record. 
I am not even sure if we still do that. I suspect that rule is mostly inactive. For testing purposes I would like to reactivate it and see if the rule gets properly processed. The bug is not bound to that rule, it could potentially be triggered by future rules as well.
So I need to test if the rule gets properly processed.
I think it is better that I test that myself.

**Comment by urapadmin** [comment]
### demo server / Kiosk 1.6.5.5
- I have patched the demo server manually (it stays 1.6.5.4). It should not have scary housekeeping messages anymore after synchronization.

**Comment by urapadmin** [comment]
but please don't close this, I want to do my test later on.

**Comment by urapadmin** [comment]
the errors are gone. Whether the qc rule is processed now properly remains to be tested.

**Comment by urapadmin** [comment]
this error still exists.

**Comment by urapadmin** [comment]
strangely does not occur on the development system, However, I have done something to address this.
### Kiosk 1.7.26.14
- lc_o_1 can be enabled again and should not lead to this warning (there might be another one coming up)

**Comment by luizaogs** [comment]
How do I test this?

**Comment by urapadmin** [comment]
you can go to quality control in the recording app (admin) and if there is a lc_o_1 rule you can switch it on and then synchronize something. And if you don't get a ton of housekeeping errors everything is fine.

Or you leave it to me

**Comment by luizaogs** [comment]
Tested in PVD, looks fine.
